### ISSUE TITLE
[Feature suggestion] WCAG trash panda mode

### ISSUE THREAD

**Issue by ericwbailey** [ericwbailey/a11y-webring.club#39]
### Feature suggestion

Until @patrickhlauke's [WCAG trash panda webring](https://mastodon.social/@patrick_h_lauke/109733347231039895) gets off the ground, this webring will have to suffice.

We should have a toggle to enable trash panda mode for the webring landing page, whatever that may be.

### Who would you like to work on this feature?

- [ ] I would like to work on this feature
- [ ] I would like the site maintainers to work on this feature

### Code of conduct

- [X] I agree to follow this project's code of conduct

**Comment by ericwbailey** [comment]
This was a cute idea, but I don't think I'll realistically get to it.
